src/drivers/github_test.go

```diff
@@ -4,15 +4,249 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
+	"testing"
 
 	. "github.com/Originate/git-town/src/drivers"
+	"github.com/stretchr/testify/assert"
 	httpmock "gopkg.in/jarcoal/httpmock.v1"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 )
 
-func GetRequestData(request *http.Request) map[string]interface{} {
+var pullRequestBaseURL = "https://api.github.com/repos/Originate/git-town/pulls"
+var currentPullRequestURL = pullRequestBaseURL + "?base=main&head=Originate%3Afeature&state=open"
+var childPullRequestsURL = pullRequestBaseURL + "?base=feature&state=open"
+var mergePullRequestURL = pullRequestBaseURL + "/1/merge"
+var updatePullRequestBaseURL1 = pullRequestBaseURL + "/2"
+var updatePullRequestBaseURL2 = pullRequestBaseURL + "/3"
+
+func TestCodeHostingDriver_CanMergePullRequest(t *testing.T) {
```

Comment:
I feel like this name should include the word Github in case we ever add tests for the other drivers

Reply:
Agreed. Done.